app/assets/javascripts/discourse/app/lib/sidebar/messages-section/message-section-link.js

```diff
@@ -0,0 +1,46 @@
+import { tracked } from "@glimmer/tracking";
+
+import { INBOX } from "discourse/components/sidebar/messages-section";
+
+export default class MessageSectionLink {
+  @tracked shouldDisplay = this._isInbox;
+
+  constructor({ group, currentUser, type }) {
+    this.group = group;
+    this.currentUser = currentUser;
+    this.type = type;
+  }
+
+  set setDisplayState(value) {
+    this.shouldDisplay = value;
+  }
```

Comment:
This will seem unnecessary now but I'm setting up the stage for the follow-up PR where I'll need this change.